### PR TITLE
Docs/tutorial workflow

### PR DESCRIPTION
**Pull request by fsuits**
## Types of changes

- \[ \] Bug fix (non-breaking change which fixes an issue)
- \[ \] New feature (non-breaking change which adds functionality)
- \[ \] Breaking change (fix or feature that would cause existing functionality to change)
- \[x] My code follows the code style of this project.
- \[ \] My change requires a change to the documentation.
- \[x] I have updated the documentation accordingly.
- \[ \] I have added tests to cover my changes.
- \[x] All new and existing tests passed.
- \[ \] All commits are signed-off.

New markdown tutorial on using trestle to split/merge oscal catalog.

### CODE REVIEW

File: docs/tutorials/python_trestle_setup.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+```
+* Ensure you have a modern [pip](https://pip.pypa.io/en/stable/installing/) (19.x or greater).
+
+```
+(venv.trestle)$ pip --version
+pip 19.2.3 from /home...
+```
+## *Install trestle*
+
+* Install [compliance-trestle](https://github.com/IBM/compliance-trestle/blob/develop/README.md).
```
Comment (butler54) [inline docs/tutorials/python_trestle_setup.md:43]: Replace with appropriate URL on the ibm.github.io/compliance-trestle website.

File: docs/tutorials/trestle_sample_workflow.md [hunk 2]
```diff
@@ -0,0 +1,693 @@
+# Tutorial: Introduction to trestle commands for manipulating an OSCAL catalog
+
+Here are step-by-step instructions to manipulate a [NIST](https://www.nist.gov/) standard [OSCAL](https://pages.nist.gov/OSCAL/) catalog using the [compliance-trestle](https://github.com/IBM/compliance-trestle/blob/develop/README.md) (`"trestle"`) tool.
+
+## *Objective*
+
+Learn how to split and merge an OSCAL catalog json file using trestle commands and its command line interface (CLI).  Trestle simplifies the manipulation and management of large OSCAL documents by allowing controlled deconstruction into smaller components, and later reconstruction after editing those components.  And all operations guarantee that the individual files conform to the corresponding OSCAL schema to maintain integrity of the document in the process.
+
+This tutorial assumes you have installed Python and installed trestle in a virtual environment per the directions found [here](./python_trestle_setup.md).
```
Comment (butler54) [inline docs/tutorials/trestle_sample_workflow.md:9]: Check to see link works in the website.

File: mkdocs.yml [hunk 3]
```diff
@@ -14,6 +14,7 @@ nav:
 - Overview: index.md
 - Trestle cli: cli.md
 - Tutorials:
+  - intro to trestle workflow: tutorials/trestle_sample_workflow.md
   - transformer construction: tutorials/task.transformer-construction/transformer-construction.md
```
Comment (butler54) [inline mkdocs.yml:18]: Missing inclusion of the python trestle setup.

**Review by butler54 — CHANGES_REQUESTED** [review]
See the changes, should be easy fixes.

File: docs/tutorials/python_trestle_setup.md (hunk 1 above)
Comment (fsuits) [reply docs/tutorials/python_trestle_setup.md]: This was in the itanium files also so I changed there, too.

File: docs/tutorials/trestle_sample_workflow.md (hunk 2 above)
Comment (fsuits) [reply docs/tutorials/trestle_sample_workflow.md:9]: confirmed it works locally after fix

File: mkdocs.yml (hunk 3 above)
Comment (fsuits) [reply mkdocs.yml]: added

**Review by butler54 — APPROVED** [review]
LGTM

**Pull request merged**